### PR TITLE
Clarify app.example.ini documentation

### PR DESCRIPTION
**Pull request by mattiasbpersson**
Hard to understand if cron i needed to activate all other or not cron.

### CODE REVIEW

**Comment by silverwind** [comment]
Please also sync changes to https://github.com/go-gitea/gitea/blob/master/docs/content/doc/advanced/config-cheat-sheet.en-us.md.

**Comment by mattiasbpersson** [comment]
Reordering docs in app.exampe.ini made diff a bit hard to read. But the order should be the same as in config-cheat-sheet.en-us.md

**Review by zeripath — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

File: custom/conf/app.example.ini [hunk 1]
```diff
@@ -994,18 +1004,7 @@ SCHEDULE = @every 24h
...
 
-[cron.update_migration_poster_id]
-; Update migrated repositories' issues and comments' posterid, it will always attempt synchronization when the instance starts.
-ENABLED = true
-; Update migrated repositories' issues and comments' posterid when starting server (default true)
-RUN_AT_START = true
-; Notice if not success
-NO_SUCCESS_NOTICE = false
-; Interval as a duration between each synchronization. (default every 24h)
-SCHEDULE = @every 24h
```
Comment (a1012112796) [inline custom/conf/app.example.ini:1005]: Why remove?

Comment (zeripath) [reply custom/conf/app.example.ini:1005]: it's just been moved higher

**Comment by 6543** [comment]
:rocket:

**Pull request merged**